Strategies/ITVision.md

```diff
@@ -0,0 +1,168 @@
+Only on exceptional, well documented and justified cases, is source code not published.
+By releasing its source code under an open source licence as well as its data under the open government licence, **ESDC's IT community is not restrained to ESDC staff only**.
+ESDC leverages crowd sourcing by including other departments, other jurisdictions, and the public to contribute back to its IT solutions.
+This [promotes canadian economic growth, skills development in Canada](https://www.theglobeandmail.com/business/commentary/article-the-essentials-for-canadas-digital-economy/), and improves trusts with canadians as well as greatly increasing ESDC's IT expertise from such broad community relationships.
+
+ESDC **publishes a large majority of the data it uses to make its decisions as open data sets under the Open Government Licence**, promoting transparency and trusts with citizens, and enabling the broader public to make use of government data for developing tools and products that contribute to their well-being and economic growth. This does not mean publishing sensitive data. Only on exceptional, well documented and justified cases, is data not published as open data.
+
+ESDC's policy and program development are done "in the open", over the Internet, enabling active participation with stakeholders whether they be citizens, 3rd party partners like banks, employers, or other jurisdictions.
+
+Technology choices **adopt open standards with preference for Open Source solutions** in order to avoid vendor lock-in, enable interoperability, and collaboration.
+Technology choices does not take a simple look as "COTS *or* Custom", rather choices look at fit for purpose which means "COTS *and* Custom": COTS for commodities and Custom for specialization. COTS is defined as per [US DoD FAQ](https://dodcio.defense.gov/Open-Source-Software-FAQ/#Q:_Is_open_source_software_commercial_software.3F_Is_it_COTS.3F)
+
+**Business Rules are seen as institutional knowledge** and are managed [internally by ESDC's staff](https://18f.gsa.gov/2018/10/09/implementing-rules-without-rules-engines/), outside of proprietary systems to avoid vendor lock-in and assure long term sustainability of Canadians core benefits systems.
+
+## 3. Digital-first and digitally enabled by design
+
+All of [ESDC's services](https://www.canada.ca/en/employment-social-development/corporate/portfolio/service-canada/programs.html) are delivered through technology with a large majority running on the public cloud to seek benefits of [rapid elasticity, broad internet access, and up to date security patching](https://doi.org/10.6028/NIST.SP.800-145). This includes up to Protected B data.
+Platform as a Service (PaaS, such as containers) and Software as a Service (SaaS) delivery cloud models ia preferred, leaving Infrastructure as a Service (IaaS) on exception basis only.
```

Comment:
Platform as a Service (PaaS, such as containers)
==>
Platform as a Service (PaaS) 

Not enirely sure "such as containers" fits there ...

Reply:
Add reference to [GC Cloud Adoption Strategy](https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/cloud-services/government-canada-cloud-adoption-strategy.html) ?
Also, according to this cloud strategy, SaaS needs to be considered first.